Documentation/how-to-debug-a-build-slave.md

```diff
@@ -137,9 +93,22 @@ over the whole lifetime of the slave (interval: 2 secs).
 
 The output file is at `/var/log/sysstat/sysstat.log`.
 
-For post mortem analysis, grab the file e.g. via scp:
+For post mortem analysis, you can find the raw sysstat.log file
+and a processed sysstat.svg file in the Artifacts section of
+each build and test job.
+
+However, if you need to retreive it while running or in a
+situation where the job does not complete, you need to grab
+the file via scp:
+
+TODO: how to make this in a nice command? docker + scp
+```
+gcloud compute ssh <build-slave>`
+sudo docker cp <container_id>:/var/log/sysstat/sysstat.log .
+exit
+gcloud compute scp <build-slave>:sysstat.log .`
+```
```

Comment:
^@olehermanse Do you have a good tip here on how to convert this into one liner?

Reply:
*does not use docker or gcloud*

Reply:
Yes, but you know "how to run ssh as scp", so with a bit of imagination you guide me on how to create a small script that can do that `docker cp` and then scp it back. :smiley:

Reply:
How about: `gcloud compute ssh <build-slave> --command 'sudo docker cp <container_id>:/var/log/sysstat/sysstat.log'` && gcloud compute scp <build-slave>:sysstat.log`?

Reply:
Nice suggestion.

However you would still need a `docker ps` with some filters inside the VM to figure out <contianer_id>, so I think I will leave it as it is...